wiki/Help_centre/Tournament_bans/id.md

```diff
@@ -0,0 +1,54 @@
+# Larangan turnamen
+
+*Laman utama: [Pusat bantuan](/wiki/Help_centre)*
+
+Larangan turnamen (*tournament ban*) merupakan bentuk hukuman yang membatasi hak seorang pemain untuk dapat berpartisipasi dalam turnamen yang didukung secara resmi, baik itu selaku pemain, penyelenggara, ataupun panitia secara umum.
+
+Larangan turnamen seringkali tidak dijatuhkan sebagai suatu hukuman yang terpisah. Meskipun demikian, pada umumnya, para pemain yang baru saja terbebas dari hukuman *[restriction](/wiki/Help_centre/Account_restrictions)* baru akan kembali dapat mengikuti turnamen setelah mereka berperilaku baik selama minimal satu tahun.
+
+Berhubung turnamen merupakan hal yang serius, seluruh hukuman larangan turnamen yang kami jatuhkan akan ditegakkan secara tegas tanpa dapat dinegosiasikan lebih lanjut.
```

Comment:
according to [ASC](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#perspective) 'kami' (we) should not be used.

Reply:
should be fixed